### ISSUE TITLE
Dark items

### ISSUE THREAD

**Issue by pintoso** [SteveTheEngineer/SS-FancyDrops#7]
idk if is possible to fix it, since the item is on the armorstand's head, etc
anyway, it is a negative point since most of the time and with most items it will happen

https://user-images.githubusercontent.com/26844778/107715763-be6f3300-6cae-11eb-949d-09aeff690d76.mp4

**Comment by pintoso** [comment]
sorry to bother you with this, i found that using `marker: false` the problem is solved

**Comment by SteveTheEngineer** [comment]
Were you using minecraft 1.16.5? I've tried it before and the items always were bright

**Comment by pintoso** [comment]
1.16.4
another time I update the server and my client to test
but as far as I researched this is a old client side problem and there was no major change in 1.16.5
>maybe you are using some different mod, etc. 
I'm using Fabric + Sodium on the client
and the server uses via version to allow 1.16.x 

**tldr;** confirmed is a problem with my client/mods, sorry for bothering you
